sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -413,8 +411,6 @@ def test_partition():
     assert partition(m).is_nonnegative
     assert partition(n).is_zero
     assert partition(p).is_positive
-    raises(ValueError, lambda: partition(x))
```

Comment:
A way to include a non-error test is to keep the x and y you had and then do
```
assert partition(x).subs(x, 7) == 15
assert partition(y).subs(x, 7) == 15
```

Reply:
Yes, I wil commit to that.